.circleci/config.yml

```diff
@@ -119,6 +119,17 @@ jobs:
           KAFKA_LISTENERS: PLAINTEXT://0.0.0.0:9092
           KAFKA_CREATE_TOPICS: gotest:1:1
           KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: "1"
+      - image: wurstmeister/zookeeper:3.4.6
```

Comment:
We are already running Kafka above (`confluentinc/cp-kafka:5.0.0`). Can we not use that instance instead? Why do we a different one?

Reply:
I tried using that one, it didn't work as expected.
When using same topic, it reads the messages from confluent test producer.
Tried with different topic, added the new topic in circleci config, got the error message saying the topic doesn't exist though I see topic is created.
Due to librdkafka requirment, I couldn't test the changes in my local.

One advantage of using a separate will remove the dependency and the unexpected issues, So I added this image.

Reply:
Let's use just one of them for both, whichever works.

Reply:
using `wurstmeister/zookeeper` and `wurstmeister/kafka` for both